internals/cli/cmd_warnings.go

```diff
@@ -28,14 +28,15 @@ import (
 	"github.com/canonical/x-go/strutil/quantity"
 
 	"github.com/canonical/pebble/client"
+	"github.com/canonical/pebble/cmd"
 	"github.com/canonical/pebble/internals/osutil"
 )
 
 const cmdWarningsSummary = "List warnings"
 const cmdWarningsDescription = `
 The warnings command lists the warnings that have been reported to the system.
 
-Once warnings have been listed with 'pebble warnings', 'pebble okay' may be used to
+Once warnings have been listed with 'warnings', 'okay' may be used to
```

Comment:
This is changing the help text in a way that IMO makes it less clear: it's not "warnings", it's "pebble warnings", a command. Once more, let's please not change the help text in this PR. Let's focus on the task at hand so that we can merge this. If there's a desire to improve the text itself anyhow, this shoud be done and discussed separately instead of inside a bulk change touching a large number of files.